cif2cell/ESPInterfaces.py

```diff
@@ -3361,3 +3362,68 @@ def __str__(self):
+# GNXAS CRYMOL FILE
+
+class CRYMOLFile(GeometryOutputFile):
+    """
+    Class for storing the geometrical data needed for outputting an input file
+    for crymol program of the GNXAS package and the method __str__ that outputs
+    the contents of the file as a string.
+    """
+
+    def __init__(self, crystalstructure, string, compound):
+        GeometryOutputFile.__init__(self, crystalstructure, string)
+        # Document string on last line after
+        self.programdoc = ""
+        self.compound = compound
+
+    def __str__(self):
+        filestring = "CRY\n"
+        filestring += self.compound+"\n"
+        if self.cell.crystal_system() == 'cubic':
+            filestring += "C\n"+str(self.cell.a)+"\n"
+        elif self.cell.crystal_system() == 'hexagonal':
+            filestring += "H\n"+str(self.cell.a)+", "+str(self.cell.c)+"\n"
+        elif self.cell.crystal_system() == 'tetragonal':
+            filestring += "E\n"+str(self.cell.a)+", "+str(self.cell.c)+"\n"
+        elif self.cell.crystal_system() == 'orthorhombic':
+            filestring += "O\n"+str(self.cell.a)+", "+str(self.cell.b)+", "+str(self.cell.c)+"\n"
+        elif self.cell.crystal_system() == 'trigonal':
+            filestring += "R\n"+str(self.cell.a)+"\n"+str(self.cell.alpha)+"\n"
+        elif self.cell.crystal_system() == 'monoclinic':
+            filestring += "M\n"+str(self.cell.a)+", "+str(self.cell.b)+", " \
+                          +str(self.cell.c)+"\n"+str(self.cell.beta)+"\n"
+        elif self.cell.crystal_system() == 'triclinic':
+            filestring += "T\n"+str(self.cell.a)+", "+str(self.cell.b)+", "+str(self.cell.c)+"\n"\
+                          +str(self.cell.alpha)+", "+str(self.cell.beta)+", "+str(self.cell.gamma)+"\n"
+        else:
+            raise SetupError("No support for " +
+                  self.crystal_system()+" crystal systems in crymol.")
+
+        filestring += "%i\n" % (self.cell.natoms())
+        index = 1
+        width = 2
+        for a in self.cell.atomdata:
+            for b in a:
+                filestring += str(index).rjust(width)+"  '"+b.spcstring().rjust(width)\
+                             +"'  "+str(b.position)+" 12 0.0 0.0\n"
+                index += 1
+        filestring += "NNY\n"
+        filestring += "1\n"
+        filestring += "'*' '*' 2.2\n"
+        filestring += "7.0\n"
+        filestring += "out   ! 3 letters name for output file\n"
+        filestring += "1,1\n"
+        filestring += "0.10\n"
+        filestring += "1\n"
+        filestring += "1\n"
+        filestring += "CIP,0. [insert Core ionization energy in Ry!]\n"
+        filestring += "3,-0.2,120.0,0.04\n"
+        filestring += "5,-1\n"
+        filestring += "'S',1,50.,15.,'S',' ','S'\n\n"
+        filestring += self.docstring
+        filestring += "!cif2cell only convert the geometrical structure, remember to change the other parameters!"
```

Comment:
Just mentioning that there are different possible approaches here - you could (a) take the parameters for a specific system (might be worth mentioning in the file) (b) remove all values for which a sensible default cannot be provided and only put suggestions in comments, ...

You will know best, I leave this up to you

Reply:
I wanted to do approach (b), but I over-did it with the default values. Made another commit, only left values where the default is good.